ietf/static_src/css/images.scss

```diff
@@ -0,0 +1,16 @@
+.circle-img {
```

Comment:
We should apply this to the homepage too, it looks to have the same size/style as those ones when checking the live site.

Reply:
They are happy to have square images, we forgot to mention that!